rules/aip0133/resource_reference_type.go

```diff
@@ -26,14 +26,20 @@ import (
 var resourceReferenceType = &lint.MethodRule{
 	Name: lint.NewRuleName(133, "resource-reference-type"),
 	OnlyIf: func(m *desc.MethodDescriptor) bool {
+		out := m.GetOutputType()
```

Comment:
nit: maybe a different variable name here? "out" makes me thing of i/o. This is the return type of an rpc? Ordinarily, I would say "outputType" or "returnMessageType", but Go likes short variables, so I'm tempted to just suggest, "ot". :/

Reply:
Sure, done!